rust/kcl-lib/tests/plane_of/program_memory.snap

```diff
@@ -18,9 +18,9 @@ description: Variables in memory after executing plane_of.kcl
       },
       "value": "Uninit",
       "xAxis": {
-        "x": 1.0,
-        "y": 0.0,
-        "z": 0.0,
+        "x": -1.0,
```

Comment:
The plane returned by the engine was left-handed, but now KCL compensates for that by fixing up the plane.